config/kubernetes-sigs/sig-api-machinery/teams.yaml

```diff
@@ -211,6 +211,7 @@ teams:
     - apelisse
     - jpbetz
     - lavalamp
+    - jiahuif
```

Comment:
@jiahuif -- the list of names should be alpha sorted.

Reply:
Thanks! fixed.

Reply:
Good catch @palnabarun, that could totally be a pre-submit check ;-)

Reply:
@apelisse -- we got that covered. :D

Ref: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/org/3174/pull-org-test-all/1480982795336552448

Reply:
Nice!